application/org.openjdk.jmc.flightrecorder.flameview/src/main/js/flameviewColoring.js

```diff
@@ -116,13 +119,33 @@ const createHslColorString = function(h,s,l){
 	return "hsl\u0028" + h + "\u002c " + s + "\u0025\u002c " + l + "\u0025\u0029";
 };
 
-const colorCell = function (d) {
-	return colorByPackage(d.data.p);
+const colorCell = function(d) {
+	if (textToSearch !== "" && (evaluateSearchElement(d.data.p) || evaluateSearchElement(d.data.n))) {
+		return "magenta";
+	} else {
+		return colorByPackage(d.data.p);
+	}
+};
+
+const evaluateSearchElement = function(text) {
+	var adjustTextToSearch = removeSpecialCharacters(textToSearch);
+	return (text !== undefined && removeSpecialCharacters(text).includes(adjustTextToSearch));
+};
+
+const removeSpecialCharacters = function(text) {
+	const map = Array.prototype.map;
```

Comment:
if the map isn't used again, why not have the next line be:
`const resultText = Array.prototype.map.call[..]` ?

Reply:
good point, i've it overlooked